### DISCUSSION TITLE
Black, white and wireless

### DISCUSSION THREAD

**Discussion by ian-thompson-yv** [george-norton/peacock#8] [Show and tell]
![img_1874_720](https://github.com/user-attachments/assets/14f745a8-4e91-438f-83c0-09d844f360ef)

**Comment by george-norton** [comment]
Very nice! FYI it is also possible to run ZMK on the Peacock. Pete has a module here:
<https://github.com/petejohanson/maxtouch-zephyr-module>

Obviously it would still be wired. But Pete has also swapped out the controller board and run the Peacock sensor wirelessly.
